api/routes.go

```diff
@@ -67,6 +67,9 @@ func addRoutes(r *gin.Engine, auth Authentication, tokenHandler TokenHandler, uc
 	router.GET("/scenarios/:scenario_id", tom, getScenario(uc))
 	router.PATCH("/scenarios/:scenario_id", tom, updateScenario(uc))
 
+	router.GET("/testrun/:testrun_id/databyscore", timeoutMiddleware(LIST_DECISION_TIMEOUT),
```

Comment:
let's put a higher timeout (say 30 sec, at least to give us some buffer while we test and release the feature)